book-app/src/app/pages/list/list.component.scss

```
@@ -0,0 +1,35 @@
+.container {
+  margin: 30px 40px 30px 40px;
```

他にも全て指定しているところがありますが↓これでもいいような…
```suggestion
  margin: 30px 40px;
```